src/test/java/org/sonarlint/intellij/issue/persistence/LiveIssueCacheTest.java

```diff
@@ -128,81 +125,60 @@ public void should_clear_store() {
   }
 
   @Test
-  @Ignore
   public void should_clear_specific_files() throws IOException {
     LiveIssue issue1 = createTestIssue("r1");
-    VirtualFile file0 = createTestFile("file0");
+    VirtualFile file0 = myFixture.copyFileToProject("foo.php", "foo0.php");
     cache.save(file0, Collections.singleton(issue1));
 
     LiveIssue issue2 = createTestIssue("r2");
-    VirtualFile file1 = createTestFile("file1");
+    VirtualFile file1 = myFixture.copyFileToProject("foo.php", "foo1.php");
     cache.save(file1, Collections.singleton(issue2));
 
     cache.clear(file1);
-    verify(store).clear("file1");
+    verify(store).clear("foo1.php");
     assertThat(cache.getLive(file1)).isNull();
     assertThat(cache.getLive(file0)).isNotNull();
   }
 
   @Test
-  @Ignore
   public void should_flush_when_requested() throws IOException {
     LiveIssue issue1 = createTestIssue("r1");
-    VirtualFile file0 = createTestFile("file0");
+    VirtualFile file0 = myFixture.copyFileToProject("foo.php", "foo0.php");
     cache.save(file0, Collections.singleton(issue1));
-    VirtualFile file1 = createTestFile("file1");
+    VirtualFile file1 = myFixture.copyFileToProject("foo.php", "foo1.php");
     cache.save(file1, Collections.singleton(issue1));
 
     cache.flushAll();
 
-    verify(store).save(eq("file0"), anyCollection());
-    verify(store).save(eq("file1"), anyCollection());
+    verify(store).save(eq("foo0.php"), anyCollection());
+    verify(store).save(eq("foo1.php"), anyCollection());
     verifyNoMoreInteractions(store);
   }
 
   @Test
-  @Ignore
   public void error_flush() throws IOException {
     doThrow(new IOException()).when(store).save(anyString(), anyCollection());
 
     LiveIssue issue1 = createTestIssue("r1");
-    VirtualFile file0 = createTestFile("file0");
-    // TODO how to link created file to module
+    VirtualFile file0 = myFixture.copyFileToProject("foo.php", "foo0.php");
     cache.save(file0, Collections.singleton(issue1));
 
     exception.expect(IllegalStateException.class);
     cache.flushAll();
   }
 
   @Test
-  @Ignore
   public void error_remove_eldest() throws IOException {
     doThrow(new IOException()).when(store).save(anyString(), anyCollection());
 
     LiveIssue issue1 = createTestIssue("r1");
-    for (int i = 0; i < 10; i++) {
-      VirtualFile file = createTestFile("file" + i);
+    for (int i = 0; i < MAX_ENTRIES_FOR_TEST; i++) {
+      VirtualFile file = myFixture.copyFileToProject("foo.php", "foo" + i + ".php");
       cache.save(file, Collections.singleton(issue1));
     }
 
     exception.expect(IllegalStateException.class);
-    cache.save(createTestFile("anotherfile"), Collections.singleton(issue1));
-  }
-
-  @Test
-  @Ignore
-  public void should_flush_on_project_closing() throws IOException {
```

Comment:
I decided to remove this test, because it would require to change the suite to an heavy test suite to emulate project closing.